src/plugins/vis_types/timeseries/public/application/components/aggs/agg.tsx

```diff
@@ -33,13 +33,13 @@ interface AggProps extends HTMLAttributes<HTMLElement> {
   siblings: Metric[];
   uiRestrictions: TimeseriesUIRestrictions;
   dragHandleProps: DragHandleProps;
-  onChange: (part: Partial<Series>) => void;
+  onModelChange: (part: Partial<Series>) => void;
```

Comment:
it was renamed due to ts error 
![image](https://user-images.githubusercontent.com/20072247/139690007-f1f4caf9-616a-43d8-8457-265ab0e353bf.png)